### PR TITLE
Randomize feature for spectra

### PR DESCRIPTION
**Pull request by goldmich**
Added `--randomize` as an optional arg for `spectra`, and added a few lines of code within the `spectra` command to randomly allocate a variant to one of the haplotypes that harbor it

### CODE REVIEW

File: mutyper/cli.py [hunk 1]
```diff
@@ -9,8 +9,10 @@
 import numpy as np
 from shutil import copyfile
 import pyfaidx
+from random import choice
 from pyliftover import LiftOver
 from Bio.Seq import reverse_complement
+import pdb
```
Comment (WSDeWitt) [inline mutyper/cli.py:15]:
```suggestion
```

File: mutyper/cli.py [hunk 2]
```diff
@@ -165,8 +167,14 @@ def spectra(args):
         spectra_data = defaultdict(lambda: np.zeros_like(vcf.samples,
                                                          dtype=int))
 
-        for variant in vcf:
-            spectra_data[variant.INFO['mutation_type']] += variant.gt_types
+        if args.randomize:
+            for variant in vcf:
+            	curr_hap = choice([x for x, y in enumerate(variant.gt_types) 
+            						 for _ in range(y)])
+            	spectra_data[variant.INFO['mutation_type']][curr_hap] += 1.
```
Comment (WSDeWitt) [inline mutyper/cli.py:174]: suggesting more descriptive variable name
```suggestion
            	random_haplotype = choice([x for x, y in enumerate(variant.gt_types) 
            						 for _ in range(y)])
            	spectra_data[variant.INFO['mutation_type']][random_haplotype] += 1.
```

**Review by WSDeWitt — COMMENTED** [review]
This looks great! Just very minor tweaks suggested

**Comment by goldmich** [comment]
looks good to me

**Review by WSDeWitt — APPROVED** [review]

**Pull request merged**